### PR TITLE
Add debug rug app

### PR DESCRIPTION
**Pull request by dhellmann**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -13,7 +13,9 @@
         'httplib2>=0.7.2',
         'python-neutronclient>=2.1',
         'oslo.config',
-        'kombu>=2.4.8'
+        'kombu>=2.4.8',
+        'webob',
```
Comment (ryanpetrello) [inline setup.py:17]: What's this about?

Comment (dhellmann) [inline setup.py:17]: On a fresh box I tried to run the new CLI after "pip install ." and had to add thse dependencies by hand. I think we've been relying on other things in devstack to install them.

File: akanda/rug/debug.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+def bandwidth_callback(self, *args, **kwargs):
+    print 'BANDWIDTH:', args, kwargs
+
+
+def debug_one_router(args=sys.argv[1:]):
+
+    main.register_and_load_opts(args)
+
+    # Add our extra option for specifying the router-id to debug
+    cfg.CONF.register_cli_opts([
```
Comment (markmcclain) [inline akanda/rug/debug.py:29]: This has been on my todo list for some time.

File: setup.py (hunk 1 above)
Comment (markmcclain) [inline setup.py:17]: Must have been getting a drive-by download of some description here previously.

**Comment by markmcclain** [comment]
LGTM
